Rakefile

```
@@ -217,6 +217,32 @@ namespace :release do
        "Groonga #{version}!!!")
     sh("git", "push", "origin", "v#{version}")
   end
+
+  desc "Post release announces"
+  task :announce do
+    latest_news = Dir.glob("doc/source/news/*.*").max do |a, b|
+      File.basename(a).to_i - File.basename(b).to_i
+    end
+    latest_release_note = File.read(latest_news).split(/^## /)[1]
+    latest_release_note_version = latest_release_note.lines.first[/[\d.]+/]
```

It seems that we can define a variable for `latest_release_note.lines.first` because it's used multiple times in this method. (If we define `latest_release_note_content` too, we can simplify `latest_release_summary` detection.)

```suggestion
    latest_release_note_title = latest_release_note.lines.first
    latest_release_note_content = latest_release_note.lines[1..-1].join
    latest_release_note_version = latest_release_note_title[/[\d.]+/]
```

---

fix: 5e90381...566dfc9 Thank you so much. We could simplify these logic!